app/controllers/api/external/v1/prosecution_conclusions_controller.rb

```diff
@@ -5,15 +5,15 @@ class ProsecutionConclusionsController < ApplicationController
         def create
           enforce_contract!
 
-          prosecution_conclusion_params["prosecutionConcluded"].map do |pc|
+          prosecution_conclusion_params["prosecutionConcluded"].each do |pc|
```

Comment:
I think this was an error, it does not make sense to have a `map` here.
I think who wrote this wanted to use an `each` loop. That's why there is a `next` inside (which I removed anyway).
By chance, it works because `map` run the code to create the array.
Fixed now.